README.md

```diff
@@ -14,3 +14,94 @@ npm start
 This project uses npm scripts to build the site to the `dist` directory.
 
 - `npm start` - Build site, watch for changes and run a hot code reloading server on http://localhost:3000
+
+## Helpers
+
+Any module dropped into the *helpers* folder will be passed in the `content` variable to all templates, with the obvious camel-case conversion (i.e. `is-equal` would be available as `content.isEqual`).
```

Comment:
Not strictly true, they are added to the root object, not to the `content` object https://github.com/tableflip/brandnewweddings/blob/master/build/jade.js#L45